node-admin/pom.xml

```diff
@@ -86,6 +86,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
Note that _vespa-athenz_ is a bundle. Should it still be included with compile scope?